src/Bindings/CMakeLists.txt

```diff
@@ -146,12 +146,12 @@ set_source_files_properties(${BINDING_OUTPUTS} PROPERTIES GENERATED TRUE)
 set_source_files_properties(${CMAKE_SOURCE_DIR}/src/Bindings/Bindings.cpp PROPERTIES COMPILE_FLAGS -Wno-error)
 
 if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
-	set_source_files_properties(DeprecatedBindings.cpp PROPERTIES COMPILE_FLAGS "-Wno-error=old-style-cast")
-	set_source_files_properties(LuaState.cpp COMPILE_FLAGS "-Wno-error=old-style-cast")
+	set_source_files_properties(DeprecatedBindings.cpp PROPERTIES COMPILE_FLAGS "")
+	set_source_files_properties(LuaState.cpp COMPILE_FLAGS "")
```

Comment:
@worktycho Do we need these lines, when setting the compile flags to empty strings? Wouldn't removing the lines altogether work the same way?